src/components/BeforeAndAfter/BeforeAndAfter.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cn from './BeforeAndAfter.module.scss';
+import teethBefore from 'assets/before_and_after/teeth-before.png';
+import teethAfter from 'assets/before_and_after/teeth-after.png';
+
+const BeforeAndAfter = () => {
+  return (
+    <>
+      <div className={cn.BeforeAndAfter}>
+        <div className={cn.Image}>
```

Comment:
Let's rename the CSS class from `Image` to `Images` since there are two pictures. Nitpicky though, up to you